packages/ember-metal/lib/computed.js

```diff
@@ -138,11 +138,12 @@ function ComputedProperty(config, opts) {
     this._getter = config;
   } else {
     assert('computed expects a function or an object as last argument.', typeof config === 'object' && !Array.isArray(config));
-    assert('Config object passed to computed can only contain `get` or `set` keys.', Object.keys(config).every((key)=> key === 'get' || key === 'set'));
-    this._getter = config.get;
+    assert('Config object passed to computed can only contain `get` and `set` keys.', Object.keys(config).every((key)=> key === 'get' || key === 'set'));
+    this._getter = config.get || function() {};
```

Comment:
I think it will be cleaner to put empty function outside of constructor, like `function K(){}` as it is done in other places, maybe it should also contain `warning` maybe that user hasn't passed getter this should be confirmed with core devs I guess

Reply:
Sure. I fix the problem in the source now. Could you take a look?
All remarks have been mentioned in the issue. Definitely worth to discuss.